lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -149,3 +154,52 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
     visibility: hidden;
   }
 }
+
+.nx-global-sidebar__other-content {
+  @include container-vertical;
+
+  padding-bottom: $nx-spacing-md;
+
+  + .nx-global-sidebar__footer {
+    border-top: 1px solid $nx-global-sidebar-separator-color;
+  }
+}
+
+.nx-global-sidebar__footer {
+  @include container-vertical;
+
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: auto;
+  padding-top: $nx-spacing-xxl;
+  padding-bottom: $nx-spacing-l;
+  text-align: center;
+}
+
+.nx-global-sidebar__support {
+  margin-bottom: $nx-spacing-xxl;
+
+  .nx-text-link {
+    align-items: center;
+    display: flex;
+  }
+}
+
+.nx-global-sidebar__support-text {
+  font-size: $nx-font-size-s;
+}
+
+.nx-global-sidebar__release {
+  font-size: $nx-font-size-xs;
+  margin-bottom: $nx-spacing-xl;
+}
+
+.nx-global-sidebar__product {
+  font-size: $nx-font-size-xs;
+  margin-bottom: $nx-spacing-xs;
+}
+
+.nx-global-sidebar__sonatype {
```

Comment:
I definitely would prefer this have "created-by" or something instead of "sonatype" in the class name.

Reply:
> I definitely would prefer this have "created-by" or something instead of "sonatype" in the class name.

I waffled back and forth between the two: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/0e6dd7b2dbf27a77c724a8c498aee6a862550936